python/doc/examples/reliability_sensitivity/reliability/plot_crossentropy.py

```diff
@@ -0,0 +1,328 @@
+physicalSpaceISResult1 = physicalSpaceIS1.getResult()
+print("Probability of failure:", physicalSpaceISResult1.getProbabilityEstimate())
+print("Coefficient of variation:", physicalSpaceISResult1.getCoefficientOfVariation())
+
+# %%
+# We can also decide to optimize only the means of the margins and keep the other parameters identical to the initial distribution.
+
+# %%
+ot.RandomGenerator.SetSeed(0)
+marginR = ot.LogNormalMuSigma(3e6, 3e5, 0.0).getDistribution()
+marginF = ot.Normal(750.0, 50.0)
+auxiliaryMarginals = [marginR, marginF]
+auxiliaryDistribution = ot.ComposedDistribution(auxiliaryMarginals)
+print("Parameters of initial distribution", auxiliaryDistribution.getParameter())
+
+# Definition of parameters to be optimized
+activeParameters = ot.Indices([0, 3])
+# WARNING : native parameters of distribution have to be considered
+bounds = ot.Interval([14, 500], [16, 1000])
+initialParameters = [15, 750]
+physicalSpaceIS2 = otexp.PhysicalSpaceCrossEntropyImportanceSampling(
+    event, auxiliaryDistribution, activeParameters, initialParameters, bounds
+)
+physicalSpaceIS2.run()
+physicalSpaceISResult2 = physicalSpaceIS2.getResult()
+print("Probability of failure:", physicalSpaceISResult2.getProbabilityEstimate())
+print("Coefficient of variation:", physicalSpaceISResult2.getCoefficientOfVariation())
+
+# %%
+# Let analyze the auxiliary output samples for the two previous simulations. We can plot initial (in blue) and auxiliary samples in physical space (orange for the first simulation and black for the second simulation).
+
+# %%
+graph = ot.Graph("Cloud of samples and failure domain", "R", "F", True, "topright")
+auxiliarySamples1 = ot.Cloud(
+    physicalSpaceISResult1.getAuxiliaryInputSample(),
+    "orange",
+    "fsquare",
+    "Auxiliary samples, first case",
+)
+auxiliarySamples2 = ot.Cloud(
+    physicalSpaceISResult2.getAuxiliaryInputSample(),
+    "black",
+    "bullet",
+    "Auxiliary samples, second case",
+)
+graph.add(initialSamples)
+graph.add(auxiliarySamples1)
+graph.add(auxiliarySamples2)
+graph.add(mycontour)
+view = viewer.View(graph)
+
+# %%
+# By analyzing the failure samples, one may want to include correlation parameters in the auxiliary distribution. In this last example, we add a Normal copula. The correlation parameter will be optimized with associated interval between 0 and 1.
+
+# %%
+ot.RandomGenerator.SetSeed(0)
+marginR = ot.LogNormalMuSigma(3e6, 3e5, 0.0).getDistribution()
+marginF = ot.Normal(750.0, 50.0)
+auxiliaryMarginals = [marginR, marginF]
+copula = ot.NormalCopula()
+auxiliaryDistribution = ot.ComposedDistribution(auxiliaryMarginals, copula)
+desc = auxiliaryDistribution.getParameterDescription()
+p = auxiliaryDistribution.getParameter()
+print(
+    "Initial parameters of the auxiliary distribution:",
+    ", ".join([f"{param}: {value:.3f}" for param, value in zip(desc, p)]),
+)
+
+# Definition of parameters to be optimized
+activeParameters = ot.Indices([0, 1, 2, 3, 4, 5])
```

Comment:
```suggestion
activeParameters = ot.Indices(6)
activeParameters.fill()
```
Another option would be:
```python
activeParameters = ot.Indices(range(6))
```
but this is less OTish.

Reply:
thanks !